Regression: failure to build on aarch64 since 1.3.2

Since https://github.com/bugsnag/panicwrap/commit/bc74a81de9a20a55de8c270a1c1e8d7b3714a7f0 (1.3.2)
the following test fails on Aarch64 on Fedora Linx Rawhide with go 1.16.4:

```
Testing    in: /builddir/build/BUILD/panicwrap-1.3.2/_build/src
         PATH: /builddir/build/BUILD/panicwrap-1.3.2/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/panicwrap-1.3.2/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags " -X github.com/bugsnag/panicwrap/version=1.3.2 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld  '"
      testing: github.com/bugsnag/panicwrap
github.com/bugsnag/panicwrap
# github.com/bugsnag/panicwrap [github.com/bugsnag/panicwrap.test]
./dup2.go:10:9: undefined: syscall.Dup2
./dup3.go:9:6: dup2 redeclared in this block
	previous declaration at ./dup2.go:9:29
FAIL	github.com/bugsnag/panicwrap [build failed]
```

--- a/dup2.go
+++ b/dup2.go
@@ -1,4 +1,5 @@
 // +build !windows
+// +build linux,!arm64
 
 package panicwrap
 

